Assets/Scripts/Managers/AudioManager.cs

```diff
@@ -0,0 +1,78 @@
+using UnityEngine.Audio;
+using System;
+using UnityEngine;
+
+
+public class AudioManager : MonoBehaviour
```

Comment:
As previously mentioned - inherit from singleton:
```suggestion
public class AudioManager : Singleton<AudioManager>
```